src/Services/Manager.vala

```diff
@@ -22,19 +22,22 @@ public class BluetoothIndicator.Services.ObjectManager : Object {
 
     public bool has_object { get; private set; default = false; }
     public bool retrieve_finished { get; private set; default = false; }
-    private Settings settings;
+    public Settings settings;
+    private DBusConnection connecting;
```

Comment:
I do not think you should use a verb for the DBusConnection name it is confusing - use the nount `connection`.

Reply:
sorry I am not fluent in speaking and writing English :D